### DISCUSSION TITLE
Equivalent to Twitter Activity API

### DISCUSSION THREAD

**Discussion by shaun0024** [mastodon/mastodon#21560] [Q&A]
Hi all,

in Twitter, there is an Activity API, where I could use it to monitor any direct messages or public tweets that mentioned me, which then sent a payload using web hooks. Is there something similar on Mastodon?

I’ve tried doing a search but could not find anything on this, possibly because I am using the wrong terminology but the closest thing I came across was the streaming api.

**Comment by trwnh** [comment]
[/api/v1/streaming/user](https://docs.joinmastodon.org/methods/streaming/#user) or you can use WebSockets (instructions on the same page)

**Reply by shaun0024** [reply]
Thank you @trwnh , I will try that out.
